src/components/votingListViewV2/votingHeaderV2.css

```diff
@@ -0,0 +1,257 @@
+    color: var(--color-slate-gray);
+    cursor: pointer;
+    display: flex;
+    justify-content: center;
+    margin-bottom: -1px;
+    padding: 14px 20px;
+    position: relative;
+
+    &:first-child {
+      border-top-left-radius: var(--border-radius-box);
+    }
+
+    &:last-child:not(.active) {
+      border-right: 0 none;
+    }
+
+    &.active {
+      background: var(--color-white);
+      border-bottom: 1px solid var(--color-white);
+      color: var(--color-ultramarine-blue);
+    }
+  }
+}
+
+.search {
+  position: relative;
+
+  & input {
+    @mixin contentNormal;
+
+    width: 177px;
+    height: 32px;
+    border-radius: 3px;
+    border: solid 1px rgba(0, 0, 0, 0.08);
+    background-color: var(--color-white);
+    outline: none;
+    margin: 0;
+    color: var(--color-blue-gray);
+    padding-left: 20px;
+    padding-right: 20px;
+    vertical-align: top;
+    transition: all ease 250ms;
+
+    &.desktopInput {
+      display: inline-block;
+    }
+
+    &.mobileInput {
+      display: none;
+    }
+
+    &:focus,
+    &.dirty {
+      & ~ .clean {
+        visibility: visible;
+      }
+    }
+  }
+
+  & .clean,
+  & .search {
+    color: rgba(0, 0, 0, 0.38);
+    position: absolute;
+  }
+
+  & .search {
+    font-size: 20px;
+    vertical-align: middle;
+    top: 4px;
+    left: 4px;
+    color: var(--color-grayscale-dark);
+  }
+
+  & .clean {
+    visibility: hidden;
+    top: 6px;
+    right: 0;
+    font-size: 16px;
+  }
+}
+
+@media (--medium-viewport) {
+  .desktopTitle {
+    display: none;
+  }
+
+  .mobileTitle {
+    display: block;
+    padding: 0;
+  }
+
+  .header {
+    & .table .tableHead li {
+      padding: 0 0 12px 12px;
+    }
+
+    &.offCanvas ~ .delegatesList {
+      & .table .tableHead {
+        width: 100%;
+        position: fixed;
+        top: var(--header-height-m);
+        left: 0;
+        z-index: 2;
+        background: var(--color-white);
+        box-shadow: 0px 0px 12px 0px rgba(0, 0, 0, 0.5);
+        padding-left: var(--tabel-header-padding-m);
+        padding-right: var(--tabel-header-padding-m);
+
+        & li {
+          padding-bottom: 0;
+        }
+      }
+    }
+  }
+
+  .filters {
+    & .filter {
+      & input {
+        width: 70px;
+        line-height: 16px;
+      }
+    }
+  }
+
+  .registerLink {
+    position: relative;
+    top: inherit;
+    left: inherit;
+  }
+
+  .search input {
+    &.desktopInput {
+      display: none;
+    }
+
+    &.mobileInput {
+      display: inline-block;
+    }
+  }
+}
+
+@media (--small-viewport) {
+  .header.offCanvas ~ .delegatesList .table .tableHead {
+    top: var(--header-height-s);
+    padding-left: var(--tabel-header-padding-s);
+    padding-right: var(--tabel-header-padding-s);
+  }
+}
+
+@media (--xSmall-viewport) {
+  .header {
+    & .account {
+      display: none;
+    }
+  }
+
+  .filters {
+    & .filter {
+      &:not(:last-of-type) {
+        margin-right: 20px;
+      }
+    }
+  }
+}
```

Comment:
Is this file duplicated?

Reply:
yes so it is easier to remove old files and replace with these

Reply:
Oh I mean, that there is the `votingHeaderV2.css` and `votingHeader.css` inside the `/votingListViewV2/` folder